### ISSUE TITLE
center columns in `to md`

### ISSUE THREAD

**Issue by a-stevan** [nushell/nushell#14552]
### Related problem

i thought it would be cool to provide some alignment support to `to md`.

### Describe the solution you'd like

if i consider the following sample table
```nushell
let t = version | select version build_time | transpose k v
```
```
#┬────k─────┬────────────v─────────────
0│version   │0.100.0
1│build_time│2024-11-21 11:32:58 +01:00
─┴──────────┴──────────────────────────
```

running `to md --pretty` on `$t` would give
```
| k          | v                          |
| ---------- | -------------------------- |
| version    | 0.100.0                    |
| build_time | 2024-11-21 11:32:58 +01:00 |
```
| k          | v                          |
| ---------- | -------------------------- |
| version    | 0.100.0                    |
| build_time | 2024-11-21 11:32:58 +01:00 |

what about a `--center` option that would take a column name and center it?
```nushell
$t | to md --pretty --center v
```
```
| k          | v                          |
| ---------- |:--------------------------:|
| version    | 0.100.0                    |
| build_time | 2024-11-21 11:32:58 +01:00 |
```

| k          | v                          |
| ---------- |:--------------------------:|
| version    | 0.100.0                    |
| build_time | 2024-11-21 11:32:58 +01:00 |

**Comment by fdncred** [comment]
I'd be fine with a center option, but it would probably need to take a cell-path or list of cell-paths so you can do more than one column

**Comment by a-stevan** [comment]
yeah a list of cell paths would probably be the most versatile! :+1:

**Comment by SkillFlame** [comment]
This sounds like something nice to have, @lazenga and I would like to give it a go

**Comment by fdncred** [comment]
Go for it

**Comment by a-stevan** [comment]
amazing, thanks @SkillFlame and @lazenga 🙏
